infrastructure/smart-contracts/scripts/tiny-keys/deployTinyKeys.mjs

```diff
@@ -62,8 +65,7 @@ async function main() {
     console.log("Got NodeLicense factory");
 
     const nodeLicenseUpgradeParams = [config.xaiAddress, config.esXaiAddress, config.chainlinkEthUsdPriceFeed, config.chainlinkXaiUsdPriceFeed, tinyKeysAirdropAddress];
-    const nodeLicense8 = await upgrades.upgradeProxy(config.nodeLicenseAddress, NodeLicense8, { call: {fn: "initialize", args: nodeLicenseUpgradeParams } });
-    await nodeLicense8.wait(BLOCKS_TO_WAIT);
+    await upgrades.upgradeProxy(config.nodeLicenseAddress, NodeLicense8, { call: {fn: "initialize", args: nodeLicenseUpgradeParams } });
```

Comment:
We still need the contract isntance for the safeVerify.

Reply:
Contract instances re-added for safe-verify!